### ISSUE TITLE
Cannot use Android setup tool since Google Play services revision 32

### ISSUE THREAD

**Issue by VitOliv** [playgameservices/play-games-plugin-for-unity#1319]
Hello,

I've been trying to update Google Play Services using the 0.9.34 version of the Unity Plugin. I've also updated my SDK recently to try to fix an error I had on my project.
However, ever since the revision of Google Play services got updated in the Android SDK, I can't seem to run the tools "Window > Google Play Games > Setup > Android setup...", neither the Jar Resolver. I have these errors in both cases:
**"System.Xml.XmlException: Document element did not appear.  Line 1, position 1."**

At first I saw this in the setup tool: 
**"Invalid classname: an exception was thrown by the type initializer GPGSDependencies.cs"**
But as I dwelved a bit deeper into the code, I found out that the sub exception also had the Xml problem.

I believe there are 3 options that might resolve this problem temporarily, but I can't seem to figure out the details for that:
- How can I revert Google Play services on the SDK back to revision 31?
- How can I change the path of the XML that the setup is trying to read?
- Are there .aar files generated elsewhere that I can use inside Unity and inject them into my project?

There could be other solutions, but right now I don't see anything else I could do. Any other sugestions are more than welcome.

**Comment by claywilkinson** [comment]
You can safely delete the xml file which contains a cached list of the resolved files.  it is located at. ProjectSettings/GoogleDependencyGooglePlayGames.xml

**Comment by VitOliv** [comment]
Just deleting it solved my issue. Thank you very much!

I didn't even know that this file was read during this process. I feel a bit "dumbed down". I was trying to update or change SDK versions because, from what I've searched on other forums like stackoverflow, the correct answer was always: "Update yout Android SDK". But in my case it didn't work of course.

For the record, that file was completely empty. I figured the problem would be something like that (an empty xml), but I would've never figured this out by myself.
